src/vm/analysis/read_only_checker/mod.rs

```diff
@@ -200,7 +199,7 @@ impl<'a, 'b> ReadOnlyChecker<'a, 'b> {
                 self.check_all_read_only(args)?;
                 Ok(false)
             }
-            Let => {
+            Let | LetStar => {
```

Comment:
Why not just replacing `let` with `let*`?

Reply:
Yeah -- I suppose we could just replace `let` with `let*`.